modules/system/assets/js/framework.js

```diff
@@ -99,10 +99,48 @@ if (window.jQuery.request !== undefined) {
 
                 delete data[inputName]
             }
+            
+            /*
+             * JavaScript Object to FormData, with support for nested objects, arrays and File objects.
+             * https://gist.github.com/ghinda/8442a57f22099bdb2e34
+             */
+            var objectToFormData = function(obj, form, namespace) {
+
+                var fd = form || new FormData();
+                var formKey;
+
+                for(var property in obj) {
+                    if(obj.hasOwnProperty(property)) {
+
+                        if(namespace) {
+                            formKey = namespace + '[' + property + ']';
+                        } else {
+                            formKey = property;
+                        }
+
+                        // if the property is an object, but not a File,
```

Comment:
Please change comment to `Properties that are objects (but not File instances) are evaluated recursively)`